src/main/groovy/com/smartbear/swagger/assertion/ValidStatusCodesAssertionCreator.groovy

```diff
@@ -0,0 +1,16 @@
+package com.smartbear.swagger.assertion
+
+import com.eviware.soapui.impl.wsdl.teststeps.RestTestRequestStep
+import com.eviware.soapui.security.assertion.ValidHttpStatusCodesAssertion
+import io.swagger.models.Response
+
+import static com.eviware.soapui.security.assertion.ValidHttpStatusCodesAssertion.LABEL
+
+
+class ValidStatusCodesAssertionCreator implements AssertionCreator {
+    @Override
+    void createAssertion(RestTestRequestStep restTestRequestStep, Map<String, Response> responseMap) {
+        ValidHttpStatusCodesAssertion assertion = (ValidHttpStatusCodesAssertion) restTestRequestStep.addAssertion(LABEL)
+        assertion.codes = responseMap.keySet().join(',')
```

Comment:
As discussed offline - we will skip the ValidStatusCodesAssertion if there is a default response specified. 
Headers are not mandatory - it just provided a list of possible headers which could be present in reponse, hence we can't add assertion on them.